rules/ensure-automount-service-account-is-false/raw.rego

```diff
@@ -0,0 +1,21 @@
+package armo_builtins
+
+# deny if default ServiceAccount automountServiceAccountToken is set to true
+deny [msga]{
+
+    sa := input[_]
+	sa.kind == "ServiceAccount"
+	sa.metadata.name == "default"
+	sa.automountServiceAccountToken == true
```

Comment:
the [default behavior in k8s](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#opt-out-of-api-credential-automounting) is that the SA token is mounted if not configured otherwise, so you would need to address the case where automountServiceAccountToken is not configured. 
also - It seems that we do have a rule that does this: "automount-default-service-account", see if it answers your needs.

Reply:
Right, changed the rule